dtrace_probe.cc

```diff
@@ -59,8 +61,17 @@ namespace node {
       return Nan::Undefined();
     }
 
+    int cblen = argsinfo.Length() - fnidx - 1;
```

Comment:
I didn't add a check for that since the only ways to get to _fire ensure that it's >= 0, but if I rework this to move this logic to the caller, then I may need to check them there.